package.json

```diff
@@ -1,30 +1,32 @@
 {
-  "private": true,
-  "scripts": {
-    "dev": "npm run development",
-    "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "watch-poll": "npm run watch -- --watch-poll",
-    "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "prod": "npm run production",
-    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
-  },
-  "devDependencies": {
-    "admin-lte": "2.3.11",
-    "axios": "^0.16.2",
-    "bootstrap": "^3.3.7",
-    "bootstrap-sass": "^3.3.7",
-    "bootstrap-tour": "^0.11.0",
-    "cross-env": "^5.2.0",
-    "font-awesome": "^4.7.0",
-    "ionicons": "^3.0.0",
-    "jquery": "^3.3.1",
-    "jquery-datetimepicker": "^2.5.20",
-    "jvectormap": "^2.0.4",
-    "laravel-mix": "^1.7.2",
-    "less": "^2.7.3",
-    "less-loader": "^4.1.0",
-    "lodash": "^4.17.10",
-    "vue": "^2.5.16"
-  }
-}
+    "private": true,
+    "scripts": {
+        "dev": "npm run development",
+        "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "watch-poll": "npm run watch -- --watch-poll",
+        "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "prod": "npm run production",
+        "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "build-waiting-lists-manager": "cd nova-components/WaitingListsManager && npm run dev",
```

Comment:
This appears to be failing when running `cannot find path specified events.js:137`

We also need to consider how we manage these moving forward. This could get _very_ messy if we just keep adding them here..